### ISSUE TITLE
Security patch releases missing

### ISSUE THREAD

**Issue by travisgroth** [tetratelabs/getenvoy-package#104]
**Describe the bug**
Envoy released `1.14.7`, `1.15.4`, `1.16.3` and `1.17.2` to address 3 high severity security issues but none of these releases are present via `getenvoy` currently.  Only `1.18.2` is showing up.

**To Reproduce**

```bash
% getenvoy list
REFERENCE                        FLAVOR       VERSION
postgres:nightly/linux-glibc     postgres     nightly
postgres:nightly/darwin          postgres     nightly
standard:1.18.2/linux-glibc      standard     1.18.2
standard:1.18.2/darwin           standard     1.18.2
standard:1.17.1/linux-glibc      standard     1.17.1
standard:1.17.1/darwin           standard     1.17.1
standard:1.17.0/linux-glibc      standard     1.17.0
standard:1.17.0/darwin           standard     1.17.0
standard:1.16.2/linux-glibc      standard     1.16.2
standard:1.16.2/darwin           standard     1.16.2
[SNIP]
```

**Expected behavior**
Patch releases going back at least a few versions be available quickly; especially after security fixes go out.

**System Information (please complete the following information):**
 - OS: MacOS, Linux

**Additional context**
N/A

**Comment by slovdahl** [comment]
Anything I can do to help get this solved? The updated versions are also missing from e.g. the APT repositories.

**Comment by karimlakhani** [comment]
1.17.3 released May 11 with fix for CVE-2021-29492 (CVSS score 8.3, High) is also missing from getenvoy.

**Comment by aalmekhlafi0** [comment]
Is there any plan to resolve this issue any time soon?
If not, how `getenvoy` community can help in such asks?

**Comment by slovdahl** [comment]
Looks like versions 1.17.3, 1.16.4, and so on exist now :+1: 

https://dl.bintray.com/tetrate/getenvoy-deb/pool/stable/g/getenvoy-envoy/

![image](https://user-images.githubusercontent.com/1417619/118753797-637d4700-b86e-11eb-9422-8f2715a3831b.png)

**Comment by lizan** [comment]
I uploaded those today, will update manifest soon so they are available to getenvoy CLI.

**Comment by lizan** [comment]
manifests are pushed as well.

**Comment by gdubicki** [comment]
What about RPMs for RHEL/Centos 7? I also don't see 1.18.3 and 1.16.4 available there (strangely 1.16.1 and 1.16.2 **are** available).

```
# yum --showduplicates list getenvoy-envoy | expand | grep -P '1\.(16|18)'
getenvoy-envoy.x86_64 1.16.0.dev.p4.g9ad964d-1p67.g2aa564b      tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p14.gee745ac-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p20.gcc64cb9-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p39.g8e006eb-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p57.gcb03985-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p58.g84affaa-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p67.g3400266-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p87.g7abb0e0-1p67.g2aa564b     tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p101.g2966597-1p67.g2aa564b    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p115.g568b759-1p67.g2aa564b    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p132.g048583b-1p67.g2aa564b    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p135.gb8996dd-1p67.g2aa564b    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p137.gb7f3561-1p67.g2aa564b    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p331.g8e34f9e-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p341.gbd40556-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p343.gd58f702-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p384.g6e91a56-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p391.gcf9b494-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p396.ga42a677-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p404.g2dcf20f-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p406.g9cfa709-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p408.g51551ae-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p416.gfbc8e57-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p426.ga1a68ab-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p434.gcbdd01c-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p459.gc003363-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p467.ga944559-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p468.g933e67a-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p476.g64731ce-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p488.g94c27aa-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p505.gc555587-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p518.g8d83ffb-1p68.g72a2636    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p519.gd3abda8-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p534.g400b3a2-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p535.g2f68705-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p552.g78bbae0-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p565.g48c2149-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p575.gf1c0031-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p585.g126a766-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p592.g02e7f64-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p594.g0b24c6a-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p596.g82e611e-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p601.g83578ec-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p619.g5fed268-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p628.g5d43118-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p642.g5d95032-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p651.g2709b6b-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p661.gf2cfbb5-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p669.g6f2ad05-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p686.gddfe3d7-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p698.gb055dd0-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p708.g7aec38f-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p718.gc8b894c-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p729.ge107c60-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p737.g51c1fa0-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p747.g012eba5-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p749.gaae6a6c-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p750.g6321e5d-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p754.g392ce7c-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p757.gb63855d7-1p69.ga5345f6   tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p760.g3b5acb2-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p762.g81b9b0e-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p778.ga536fd8-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p791.g2550806-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p798.gc318156-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.0.dev.p833.g8888ee7-1p69.ga5345f6    tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.1.p0.g0717f49-1p70.g486d4e2          tetrate-getenvoy
getenvoy-envoy.x86_64 1.16.2.p0.ge98e41a-1p70.g486d4e2          tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p3.gdcf8340-1p71.gbe6132a      tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p12.gefe73e8-1p71.gbe6132a     tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p33.gb794aea-1p72.g28ef262     tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p47.g38a6fe5-1p72.g28ef262     tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p96.g8814014-1p74.gbb8060d     tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p101.g274e883-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p106.g268b94b-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p108.g2ee9543-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p114.g5899f35-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p117.gfc461bb-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p125.gf6679d5-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p133.g8e2286b-1p74.gbb8060d    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p383.ge20957c-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p385.gd28f2f6-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p387.g7277b55-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p388.gdf3f298-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p397.g031f75d-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p404.g73eec64-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p418.gf45181d-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p432.g63c5703-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p446.g9a89e31-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p454.g48c4fb8-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p456.ga240824-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p468.g362d6de-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p479.gb0ce15c-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p490.g38f6738-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p504.g2eb6359-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p505.gfcdaf20-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p509.g486cd88-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p515.g8e8a21d-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p525.g20e4dd2-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p535.g1134a16-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p546.g45e91d3-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p553.g2700a96-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p560.ga45ce65-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p573.g0b02533-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p582.ga67d8e5-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p593.g9c6a44c-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p599.g56d3f8a-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p601.g4ecf3f7-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p604.g50c8be5-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p615.g8771f7f-1p75.g76c310e    tetrate-getenvoy
getenvoy-envoy.x86_64 1.18.0.dev.p625.g432cc06-1p75.g76c310e    tetrate-getenvoy
```
